zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/ZeebeRocksDbFactory.java

```diff
@@ -56,31 +57,50 @@ public ZeebeRocksDbFactory(
 
   @Override
   public ZeebeTransactionDb<ColumnFamilyType> createDb(final File pathName) {
-    final ZeebeTransactionDb<ColumnFamilyType> db;
     final List<AutoCloseable> closeables = Collections.synchronizedList(new ArrayList<>());
```

Comment:
❓ Do you remember why this is actually a synchronized list 🤔

Reply:
No idea, I just copied it and assumed we have a good reason for it :smile: Possibly it's related to creating transaction/transaction contexts from possibly many threads? :shrug: